### PR TITLE
ADD Btcc

### PR DESCRIPTION
**Pull request by Swingcloud**

### CODE REVIEW

File: lib/cryptoexchange/exchanges/btcc/services/pairs.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          adapt(output)
+        end
+
+        def adapt(output)
+          market_pairs = []
+          output.each do |pair|
+            market_pairs << Btcc::Models::MarketPair.new(
+                              base: pair[:base],
+                              target: pair[:target],
+                              market: MARKET::NAME
```
Comment (dominicwong617) [inline lib/cryptoexchange/exchanges/btcc/services/pairs.rb]: @Swingcloud same comment as ANX's PR

**Comment by dominicwong617** [comment]
@Swingcloud you have a conflict

**Comment by Swingcloud** [comment]
let me check @dominicwong617

**Comment by dominicwong617** [comment]
@tmlee merging this

**Pull request merged**